requirements/app/test.txt

```diff
@@ -4,11 +4,13 @@ pytest-timeout ==2.1.0
 pytest-cov ==4.0.0
 pytest-doctestplus >=0.9.0
 pytest-asyncio ==0.20.3
+pytest-random-order <=1.1.0
```

Comment:
Tesst requirements can and should pin to the latest version

Instead of `<=`

Reply:
```suggestion
pytest-random-order ==1.1.0
```